static/directives/header-bar.html

```diff
@@ -44,7 +44,7 @@
 
         <!-- Normal -->
         <ul class="nav navbar-nav navbar-right hidden-xs" ng-switch on="user.anonymous">
-          <li class="new-ui-toggle" ng-if="Features.UI_V2">
+          <li class="new-ui-toggle" ng-if="Features.UI_V2 && checkElementByID('signin-username') !== true">
```

Comment:
you don't need the `!== true` it's already a boolean just the return value will suffice

Reply:
Fixed